### PR TITLE
add roadmap for Q1

### PR DESCRIPTION
**Pull request by Slashek**

### CODE REVIEW

File: marketplace_builder/views/pages/roadmap/2019-q1/module-templates.liquid [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* It handles error checking in values provided for rendering
+* It has optional HTML escaping build in
+
+### Examples
+
+Given a template like this:
+
+```yaml
+---
+slug: (*| &blog_root |*)
```
Comment (pavelloz) [inline marketplace_builder/views/pages/roadmap/2019-q1/module-templates.liquid]: I think you you copied deprecated syntax in here

**Comment by diana-lakatos** [comment]
Reviewed, edited. (Grammarly checks done for all topics.)

File: marketplace_builder/views/pages/roadmap/2019-q1/module-templates.liquid [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+## Solution
+
+A solution that is partially implemented uses the Marketplace Kit (Partner Portal internally uses the Marketplace Kit for module deployment, so internally it's commonplace). In Marketplace Kit, there is a piece of code run on the file before it gets synced or deployed that handles additional layer of templating.
+
+### Technical details
+
+The implementation uses mustache.js which is a lib with **logicless** templates for JavaScript. Some pros of this option:
+
+* The code is based on a descending parser and quite solid. 
+* Tags used can be configured (examples below use `(*|` and ` |*))`.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/roadmap/2019-q1/module-templates.liquid]: outdated

File: marketplace_builder/views/pages/roadmap/index.liquid [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+This section contains information about features we are implementing or planning to implement.
+
+### Roadmap
+
+## Q1 2019
+
+We aim to deliver the following features and enhancement by the end of Q1 2019.
+
+* [Liquid tags for basic CRUD operations](/roadmap/2019-q1/liquid-tags-for-basic-crud-operations)
+* [Durable storage for simple key => value pairs](/roadmap/2019-q1/durable-storage-for-key-value-pairs)
```
Comment (pavelloz) [inline marketplace_builder/views/pages/roadmap/index.liquid]:
```suggestion
* [Persistent storage for simple key => value pairs](/roadmap/2019-q1/durable-storage-for-key-value-pairs)
```

**Comment by diana-lakatos** [comment]
Reviewed and edited two additional topics.

**Review by pavelloz — APPROVED** [review]

**Pull request merged**